extension/content/content.js

```diff
@@ -5,20 +5,29 @@ onLocationHrefChange(() => {
     main()
 })
 
-function main() {
+async function main() {
```

Comment:
If we encapsulate `fetchTimeComments` in `fetchChapters` no changes are required here.